spec/javascript/test/containers/AirportReviewContainer.js

```diff
@@ -0,0 +1,47 @@
+import '../../testHelper.js'
+import fetchMock from 'fetch-mock'
+
+import AirportReviewContainer from '../../../../app/javascript/react/containers/AirportReviewContainer'
+import AirportReviewTile from '../../../../app/javascript/react/components/AirportReviewTile'
+
+describe('AirportReviewContainer', () => {
+  let reviews,
+      wrapper;
+
+  beforeEach(() => {
+    jasmineEnzyme();
+    reviews = {
+      reviews: [
+        {"id":1,"title":"This is a title","body":"This body has to be at least twenty chars","overall_rating":5,"queue_time":4,"cleanliness":3,"wifi":2,"staff":1,"lounge_space":3,"airport_id":1,"created_at":"2019-04-24T14:16:16.896Z","updated_at":"2019-04-24T14:16:16.896Z"}
```

Comment:
Please separate hashes with many or long key/value pairs onto multiple lines:
```suggestion
      {
        "id":1,
        "title":"This is a title",
        "body":"This body has to be at least twenty chars",
        "overall_rating":5,
        "queue_time":4,
        "cleanliness":3,
        "wifi":2,
        "staff":1,
        "lounge_space":3,
        "airport_id":1,
        "created_at":"2019-04-24T14:16:16.896Z",
        "updated_at":"2019-04-24T14:16:16.896Z"
      }
```

Reply:
here and elsewhere!